front/app/components/Author/index.tsx

```diff
@@ -1,5 +1,4 @@
-import React, { PureComponent } from 'react';
-import { adopt } from 'react-adopt';
+import React, { memo } from 'react';
```

Comment:
Refactor to functional component + moved UserName into the render

Reply:
Nice that you refactored it, it makes it so much more understandable! 🚀 
Do you think you can also add some basic tests to it?

Reply:
We'd like to do the release this afternoon so I'm going to merge it like this and I put adding unit tests on my to do (I still have to learn unit tests, will happen in one of the next 10% moments). I will assign you again when I've added them in a separate PR sometime in the coming weeks. :)